db/schema.rb

```diff
@@ -134,7 +134,7 @@
     t.string "additional_gcse_equivalencies"
     t.boolean "can_sponsor_skilled_worker_visa", default: false
     t.boolean "can_sponsor_student_visa", default: false
-    t.string "campaign_name", default: "no_campaign"
+    t.integer "campaign_name"
```

Comment:
Don't we need another migration to drop column and create column as the data type has changed?

Reply:
done